include/aie/Dialect/AIEX/IR/AIEX.td

```diff
@@ -518,6 +519,36 @@ def AIE_IpuDmaMemcpyNdOp: AIEX_Op<"ipu.dma_memcpy_nd", [
   let hasVerifier = 1;
 }
 
+def AIE_IpuDmaWaitOp: AIEX_Op<"ipu.dma_wait", []> {
+  let summary = "Blocking operation to wait for a DMA to complete execution.";
+  let description = [{
+    The IpuDmaWaitOp blocks until the DMA referenced through `symbol` completes execution
+    and issues a task-complete-token. NOTE(jornt): do we want to support synchronization
```

Comment:
drive-by comments (will review thoroughly soon): I believe this note should be removed?

Reply:
Yeah, I just wanted to bring this up. Is there any polling support right now or any interest in adding that?

Reply:
I don't know 100% but just yesterday I was discussing this with @benvanik - there is indeed interest and I will likely figure the how/when/where/why soon.